apps/settings/ws.py

```diff
@@ -222,12 +223,17 @@ def run_import_user(self, data):
                 msg = _('No LDAP user was found')
             else:
                 orgs = self.get_orgs(org_ids)
-                new_users, error_msg = LDAPImportUtil().perform_import(users, orgs)
+                is_sync_all = '*' in username_list
+                new_users, error_msg, disable_users = LDAPImportUtil(
+                    self.category, is_sync_all
+                ).perform_import(users, orgs)
                 ok = True
                 success_count = len(users) - len(error_msg)
                 msg = _('Total {}, success {}, failure {}').format(
                     len(users), success_count, len(error_msg)
                 )
+                if disable_users:
+                    msg += _(', disabled {}').format(len(disable_users))
                 self.set_users_status(users, error_msg)
         except Exception as e:
             msg = str(e)
```

Comment:
There should be no other changes made to the code.

This message provides nothing new and merely states that there is no need for verification at this point since there were no previous changes.